.github/workflows/integration.yml

```diff
@@ -71,3 +71,19 @@ jobs:
         with:
          files: ./e2e-coverage.out
          verbose: true
+      
+      - name: Run GitLab E2E  #using retry because the GitHub token is being throttled.
+        uses: nick-invision/retry@943e742917ac94714d2f408a0e8320f2d1fcafcd
+        env:
+          GITLAB_AUTH_TOKEN: ${{ secrets.GITLAB_AUTH_TOKEN }}
+        with:
+          max_attempts: 3
+          retry_on: error
+          timeout_minutes: 30
+          command: make e2e-gitlab
+
+      - name: codecov
+        uses: codecov/codecov-action@81cd2dc8148241f03f5839d295e000b8f761e378 # 2.1.0
+        with:
+         files: ./e2e-coverage.out
+         verbose: true
```

Comment:
Not needed? Seems like GitLab e2e tests don't output this file.

Reply:
Subsequent PRs will add tests for this.